src/components/ui/CoralTab.js

```diff
@@ -0,0 +1,56 @@
+import React, { Component,  PropTypes } from 'react';
+import Radium from 'radium';
+
+/**
+ * Tab
+ *
+ * Material Design Reference: https://github.com/google/material-design-lite/tree/mdl-1.x/src/tabs
+ */
+@Radium
+export default class CoralTab extends Component {
+
+  static propTypes = {
+    active: PropTypes.bool,
+    className: PropTypes.string,
+    onClick: PropTypes.func,
+    tabId: PropTypes.number,
+    style: PropTypes.object
+  };
+
+  render() {
+    const { active, className, children, onClick, style, tabId, ...rest } = this.props;
+    return (
+      <a
+        style={[
+          styles.base,
+          active ? styles.active : {},
```

Comment:
this can be shortened to `active && styles.active` which is what we're doing elsewhere in the app. you can pass `undefined` or `false` to an array of styles and nothing happens.